frontend/src/ui/player/index.tsx

```diff
@@ -118,16 +118,38 @@ const delayTill = (date: Date): number => {
 export const InlinePlayer: React.FC<PlayerProps> = ({ className, event, ...playerProps }) => {
     const aspectRatio = getPlayerAspectRatio(event.syncedData.tracks);
     const isDark = useColorScheme().scheme === "dark";
+    const ref = useRef<HTMLDivElement>(null);
+
+    const minControlWidth = 470;
+    const [controlsFit, setControlsFit] = useState<boolean>(true);
+
+    const resizeObserver = new ResizeObserver(() => {
+        if (ref.current) {
+            const scaling = ref.current.offsetHeight / aspectRatio[1];
+            const inferredWidth = scaling * aspectRatio[0];
```

Comment:
I'm really really nitpicky now, but to me, this feels more natural:

```suggestion
            const inferredWidth = ref.current.offsetHeight * (aspectRatio[0] / aspectRatio[1]);
```